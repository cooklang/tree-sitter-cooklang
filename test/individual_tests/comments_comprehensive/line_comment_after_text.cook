Cook slowly -- patience is key