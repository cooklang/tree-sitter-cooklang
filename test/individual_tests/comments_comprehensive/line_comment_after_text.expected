(recipe
  (step
    (plain_text)
    (comment)))